app/views/thredded/posts/_content_field.html.erb

```diff
@@ -1,4 +1,6 @@
 <li>
   <%= form.label :content, content_label %>
+  <%= render 'thredded/posts/before_content_field', form: form %>
   <%= form.text_area :content, { rows: 5, required: true }  %>
+  <%= render 'thredded/posts/after_content_field', form: form %>
```

Comment:
I would say this should be just "after_content" (not "after_content_field"), since it's not a given that a field will be rendered there, and that this should be documented, but since this is an interim solution, LGTM anyway.